store/sqlstore/post_store.go

```diff
@@ -1247,7 +1247,7 @@ func (s *SqlPostStore) determineMaxPostSize() int {
 				table_name = 'posts'
 			AND	column_name = 'message'
 		`); err != nil {
-			l4g.Error(utils.T("store.sql_post.query_max_post_size.error") + err.Error())
+			mlog.Error(utils.T("store.sql_post.query_max_post_size.error") + err.Error())
```

Comment:
translation and concatenation instead of Sprintf

Reply:
Get this on another pas (it works)